src/Trains.NET.Rendering/Game/Game.cs

```diff
@@ -148,21 +139,20 @@ public void DrawFrame()
 
                 using IImageCanvas imageCanvas = _imageFactory.CreateImageCanvas(_width, _height);
 
+                imageCanvas.Canvas.Save();
                 RenderFrame(imageCanvas.Canvas, pixelMapper);
-
-                _imageCache.Set(this, imageCanvas.Render());
+                imageCanvas.Canvas.Restore();
 
                 foreach (IScreen screen in _screens)
                 {
-                    if (_imageCache.IsDirty(screen))
-                    {
-                        _screenDrawTimes[screen].Start();
-                        using IImageCanvas screnCanvas = _imageFactory.CreateImageCanvas(_screenWidth, _screenHeight);
-                        screen.Render(screnCanvas.Canvas, _screenWidth, _screenHeight);
-                        _screenDrawTimes[screen].Stop();
-                        _imageCache.Set(screen, screnCanvas.Render());
-                    }
+                    _screenDrawTimes[screen].Start();
+                    imageCanvas.Canvas.Save();
+                    screen.Render(imageCanvas.Canvas, _screenWidth, _screenHeight);
```

Comment:
Should we keep the cache for these, in case it helps in future? If not, probably should get rid of the Changed event, and stop listening to it.